pkg/daemon/tasks.go

```diff
@@ -53,12 +55,64 @@ func (d *Daemon) listTasksHandler(engine api.Engine) func(w http.ResponseWriter,
 			return
 		}
 
+		cr, _ := engine.RunnerByName("cluster:k8s")
```

Comment:
This whole handler will need to handle the `view` of the tasks (i.e. the tasks dashboard) properly soon, but for now I just printout an HTML table. It is out of scope for this PR to address the UI properly.